vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,82 @@ sidebarDepth: 2
-## Create the Project Structure
-First, create a parent project directory (e.g. `example-bundle`) along with a child bundle directory. In a project generated by the [Entando Component Generator](../../../docs/create/component-gen-overview.md), the microservice and micro frontend source files live under the parent directory.
+## Initialize a Bundle Project
 
-``` sh
-mkdir -p example-bundle/bundle; cd example-bundle/bundle
-```
+1. Create the bundle structure by scaffolding default files and folders:
+   ``` sh
+   ent bundle init YOUR-BUNDLE-NAME
+   ```
 ## Add a Simple Widget
 
-Create a widget directory
-``` sh
-mkdir widgets
-```
-
-Create a widget descriptor file within that directory
-``` sh
-touch widgets/example-widget.yaml
-```
-
-Populate the widget descriptor file `example-widget.yaml` with a simple definition. Make sure to retain the correct YAML indentation of 2 or 4 spaces.
-``` yaml
-code: example-widget
-titles:
-   en: Example Widget
-   it: Widget d'esempio
-group: free
-customUi: <h2>Hi from Example Widget</h2>
-```
-
-## Create the Bundle Descriptor
-
-The main file processed by the Entando Component Repository is `descriptor.yaml`, which describes all of the components within the bundle. The name of the bundle descriptor file must be `descriptor.yaml` and it must be stored in the child bundle directory (e.g. `example-bundle/bundle`).
-```sh
-touch descriptor.yaml
-```
-
-Populate the bundle descriptor file with the following YAML definition
-``` yaml
-code: example-bundle
-description: This is an example of an Entando bundle
-components:
-  widgets:
-    - widgets/example-widget.yaml
-```
-Component descriptor file names and locations (e.g. `widgets/example-widget.yaml`) are arbitrary since the bundle descriptor explicitly points to those files. Convention is to group components by type with all widgets in one directory, all page templates in another, etc.
+1. From the bundle root folder, create and enter a widget directory:
+   ``` sh
+   mkdir -p platform/widgets
+   ```
+   Descriptor files are organized by component type, with all micro frontends, microservices and platform-specific components located under their respective parent folders. 
+   
+   The parent folder "platform" contains the child folder "widgets" in which to store the widget descriptor file.
+   OR
+   Widgets are a specific type of platform component, so the "platform" directory includes a child directory dedicated to widget files. 
+
+2. Create a widget descriptor file within the widget directory:
+   ``` sh
+   touch platform/widgets/example-widget.yaml
+   ```
+   A corresponding top-level bundle descriptor file is not necessary.
+
+   Platform-specific components use descriptor file names and locations (e.g. `platform/widgets/example-widget.yaml`) which are arbitrarily and explicitly defined within the bundle descriptor (`entando.json`). Convention is to group components by type with all widgets in one directory, all page templates in another, etc.
```

Comment:
Is this line true? I thought we did not need to list the specific platform components at all in the entando.json but that they would automatically be included. Maybe I'm imagining that.

Reply:
@nshaw i actually don't know without an example to reference. i keep talking myself into it being either true or false... i'll keep playing to hopefully figure it out definitively

Reply:
@Lyd1aCla1r3 me too. We need to test it. I think/hope the latest CLI will let you try it out... Maybe the demo tomorrow will also help, or we can ask if there's time.